tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/display.py

```diff
@@ -9,7 +9,7 @@
 
 class Display(Base):
 
-    _page_locator = (By.ID, 'sound')
```

Comment:
It can show git info. I don't remember if I installed a plugin or not for that. The steps I did were:
1. highlight the line I'm interested in
2. right click -> git history -> show history for selection